hybrasyl/Objects/Monster.cs

```diff
@@ -627,19 +266,23 @@ public void AssailAttack(Direction direction, Creature target = null)
                 //try to get the creature we're facing and set it as the target.
             }
             
-             Attack(direction, target);
-                
-            
+            // A monster's assail is just a straight attack, no skills involved.
+            SimpleAttack(target);
+                            
             //animation handled here as to not repeatedly send assails.
             var assail = new ServerPacketStructures.PlayerAnimation() { Animation = 1, Speed = 20, UserId = this.Id };
-            var sound = new ServerPacketStructures.PlaySound() { Sound = (byte)1 };
             //Enqueue(assail.Packet());
             //Enqueue(sound.Packet());
             SendAnimation(assail.Packet());
-            PlaySound(sound.Packet());
+            PlaySound(1);
         }
 
-
+        /// <summary>
+        /// A simple directional attack by a monster (equivalent of straight assail).
+        /// </summary>
+        /// <param name="direction"></param>
+        /// <param name="target"></param>
+        public void SimpleAttack(Creature target) => target?.Damage(_simpleDamage, OffensiveElement, Enums.DamageType.Physical, this);
```

Comment:
was damage updated to account for element handling?

Reply:
No, not yet. I'll make a note to handle that.